### PR TITLE
Sysadmin to SRE

### PR DESCRIPTION
**Pull request by Duffney**
# About

This is a post about...

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

**Review by wordlee — APPROVED** [review]
Great post @Duffney! I've done an initial pass through the text, I didn't make many edits, though I did introduce a bit more whitespace in the text, you had quite a lot of similarly sized paragraphs and that can be a bit daunting when readers do their initial scan of the text.

**Review by robpearson — COMMENTED** [review]
Overall, I think this is a great article. I added a couple of comments for your review. :)

File: blog/2020-09/sysadmin-to-sre/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Needless to say, DevOps is hot!
+
+In more recent years a new title has emerged from Google, Site Reliability Engineer (SRE). SRE is not yet as popular as its predecessor, but it is often used synonymously by recruiters. With all the hype you have to wonder, are they worth it? And if so, how do you get there? Looking at job postings will likely just confuse you. Everything from 10 years of C# experience to intimate knowledge of Active Directory domain trusts are listed in the job descriptions. With such a broad set of requirements, where do you even begin?
+
+Start by understanding that DevOps and SRE are advanced job roles. You can start a career in web development by applying for associate-level positions and system administration at the help desk. DevOps and SRE don’t yet have that entry position. Typically a DevOps practitioner has already spent a few years working in technology. This is because DevOps and SRE roles require an understanding of development and infrastructure. It’s difficult to learn both disciplines at once, which is why most practitioners have an existing background in one or the other.
```
[the remark this answers is not in the document]
Comment (wordlee) [reply blog/2020-09/sysadmin-to-sre/index.md:25]: I think it's a fairly common term. I don't think it's quite entry-level, but they're fairly close in meaning.

File: blog/2020-09/sysadmin-to-sre/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+All the advantages of Git are not immediately apparent. Focus on the value it adds upfront vs. in the future. That immediate value is auditable code through a commit history, increased collaboration, and easier management than a file share. Most importantly though, source control unlocks everything else.
+
+## Pull requests mean deployments
+
+Tooling that has traditionally been administered by release engineers has become table stakes for anyone writing code. 
+
+Yes, scripts are code too. 
+
+Release engineering is a sub-discipline of software engineering which focuses on the compilation, assembly, and delivery of source code. There is a lot you don’t need to know about release engineering, but two types of tools exist that you won’t want to live without; continuous integration and continuous deployment collectively known as CI/CD.
```
[the remark this answers is not in the document]
Comment (Duffney) [reply blog/2020-09/sysadmin-to-sre/index.md:65]: I have no objection. Make sense to me.

File: blog/2020-09/sysadmin-to-sre/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Duffney) [reply blog/2020-09/sysadmin-to-sre/index.md:25]: I'm good with either term. Each delivers the right context for my point.

**Review by robpearson — APPROVED** [review]

**Pull request merged**